aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -68,6 +68,8 @@ Tensor& _clamp_out_cpu(
     _th_clamp_max_out(result, self, *max);
   } else if (min) {
     _th_clamp_min_out(result, self, *min);
+  } else {
```

Comment:
I am curious, if `min` and `max` both are None, shouldn't clamp give back the `result` (which is the original tensor passed in)?

Reply:
I think either of the two following approaches are valid:
- torch.clamp(tensor, min=None, max=None) gives back the same tensor (what you are proposing)
- torch.clamp(tensor, min=None, max=None) errors out (what is implemented in this PR and in previous versions of pytorch).

Right now torch.clamp(tensor, min=None, max=None) returns an empty tensor, which is definitely a bug. I'm restoring the previous semantics of clamp (to error out in this case) for the following reasons:
1) I'm worried the user might have made a mistake because this function becomes a no-op
2) torch.clamp(tensor, ...) used to always return a new tensor. If we have it return the same tensor for `min, max=None`, that subtly changes the semantics and alias information.

Please feel free to think more about this problem

Reply:
that sounds good to me :)